python/ray/_private/runtime_env/working_dir.py

```diff
@@ -214,6 +220,11 @@ def modify_context(
             ]
         set_pythonpath_in_context(python_path=str(working_dir), context=context)
 
+        # Use placeholder here and will replace it by `pre_worker_startup`.
+        context.job_dir = WorkingDirPlugin.job_dir_placeholder
```

Comment:
The `context.job_dir` is also assigned at `pre_worker_startup`, why it need init here too?